lib/resolvers.js

```diff
@@ -147,6 +229,21 @@ const Resolvers = {
 
 module.exports = Resolvers;
 
+async function validateAdminAccess (request) {
```

Comment:
It would be nice if you could add this validation at the route configuration level, not in each handler. If you need multiple auth strategies on a route, you can use [`nuisance`](https://www.npmjs.com/package/nuisance), although it needs to be updated for hapi 17.

Reply:
I forgot we only had one route in GraphQL :-D